kiwi/config/functions.sh

```diff
@@ -263,7 +263,7 @@ function baseSetupUserPermissions {
         fi
         if [[ ! "${shell}" =~ nologin|true|false ]];then
             group=$(grep "${group}" /etc/group | cut -f1 -d:)
-            chown -c -R "${usern}:${group} ${dir}/*"
+            chown -c -R ${usern}:${group} ${dir}/*
```

Comment:
Spaces can be present here. I would suggest `chown -c -R "{usern}:${group}" "${dir}"/*`

Reply:
Good point